Framework/Algorithms/src/StripPeaks.cpp

```diff
@@ -253,11 +253,11 @@ StripPeaks::removePeaks(API::MatrixWorkspace_const_sptr input,
                           << " x + " << a2 << " x^2\n";
     }
 
-    // Loop over the spectrum elements
+	
     const int spectrumLength = static_cast<int>(Y.size());
     for (int j = 0; j < spectrumLength; ++j) {
       // If this is histogram data, we want to use the bin's central value
```

Comment:
That comment was describing the computation based on `X`, which is now gone, so can be removed as well.